.circleci/workflows.yml

```diff
@@ -573,13 +577,31 @@ workflows:
           requires:
             - compile-contracts
             - create-relayers
-      - build-and-publish-container-image:
+      - test-container-image:
           requires:
-            - test-relayer
-            - e2e-test-ethereum
-            - e2e-test-optimism
-            - e2e-test-arbitrum
-            - e2e-test-polygon-zkevm
+            - compile-contracts
+            - create-relayers
```

Comment:
Here we have that `test-container-image` depends on `compile-contracts`, but if you look at the definition of `test-container-image`, you will see we already compile contracts there:
```
  test-container-image:                                         
    executor: vm                                                
    environment:                                                
      IMAGE_NAME: ghcr.io/beamer-bridge/beamer                  
    steps:                                                      
      - initialize-environment                                  
      - run:                                                    
          name: Build a beamer image                            
          command: make container-image IMAGE_NAME=${IMAGE_NAME}
      - run:                                                    
          name: Compile contracts                               
          command: ape compile                                  
```

Can this then be removed from the job?